### PR TITLE
add ya and yublf filters for private use

### PR DESCRIPTION
**Pull request by adon-at-work**
- add ya filter
- update frCoPt() to properly handle null and undefined
- add test cases

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: tests/unit/private-xss-filters.js [hunk 1]
```diff
@@ -363,9 +380,33 @@ Authors: Nera Liu <neraliu@yahoo-inc.com>
...
     });
 
+    describe("private-xss-filters: utility tests", function() {
+        it('htmlDecode d exists', function() {
+            expect(filter.d).to.be.ok();
+        });
+        it('htmlDecode d test', function() {
+            expect(filter.d(null)).to.equal('null');
+            expect(filter.d()).to.equal('undefined');
+            expect(filter.d('&Aacute;&#0;&#x0D;&#x80;&#x82;&#x94;&#x9F;&#xD800;&#xFDD0;')).to.equal('&Aacute;\uFFFD\uFFFD\u20AC\u201A\u201D\u0178\uFFFD\uFFFD');
```
Comment (neraliu) [inline tests/unit/private-xss-filters.js]: filter.d is not the full html decoder, right?

Comment (adon-at-work) [inline tests/unit/private-xss-filters.js]: just a simple one. not full

**Comment by adon-at-work** [comment]
renamed `yufbl` to `yublf`

**Pull request merged**